apps/app/src/pages/search/[...params]/index.tsx

```diff
@@ -237,25 +348,53 @@ const SearchResults = () => {
 					</Text>
 				</Box>
 			)}
+
+			{showStateAlertMessage && (
+				<Box className={classes.banner}>
+					<Text variant={variants.Text.utility1white}>
+						<Trans
+							i18nKey='alerts.search-page-legislative-map'
+							ns='common'
+							components={{
+								ATLink: (
+									<Link
+										external
+										variant={variants.Link.inheritStyle}
+										href='https://www.erininthemorning.com/p/anti-trans-legislative-risk-assessment-96f'
+										target='_blank'
+									></Link>
+								),
+							}}
+						/>
+					</Text>
+				</Box>
+			)}
+
```

Comment:
Can we extract the banner code out in to a separate component?

Reply:
Yes good call, I'll work on that this week!

Reply:
Would the banners go in the story book as components?

Reply:
Yeah, you can create it in `components/core` in the ui package - there are plenty of `*.stories.tsx` files to use as a starting template.

Reply:
Moved the banners to components although my folder structure may be a bit different, was having an issue with imports

Reply:
I fixed part of it - you were trying to move the component in to a storybook file (`*.stories.tsx` shouldn't get imported in to any functional part of the app - those are for Storybook only). I moved the component to `ui/components/core/LocationBasedAlertBanner/index.tsx` and set up 3 different use examples. Inside the storybook file, in the `RenderWrapper` component I put in there (just for purposes of pulling mock API data), you'll see how the API call is done. The API can return more than one result, in the event that there are multiple alerts for an area or overlapping alerts

Reply:
I didn't touch the implementation on this page though -- that still needs to be done